docs/best-practice/limit-and-offset-vs-cursor-api-and-hasmore.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 50
+title: LIMIT and OFFSET vs Cursor API and hasMore
+---
+
+Queries that return large volumes of data may require more than one response to provide the complete results. A common method is using the `LIMIT` and `OFFSET` statements in the query. However, this is not the optimized approach. When these options are used, each time the query is invoked query processing is performed. 
+
+A better approach is to use the `cursor` API.  The response from the cursor API call contains a boolean attribute, `hasMore`. If `hasMore` is true, the next batch of records will be ready on the server. In subsequent calls, the records are returned from the last position. 
+
+The `batchSize` parameter can be set in the `cursor` API to configure the number of documents returned per request. The `batchSize` default is `100` and the maximum value is `1000`. 
+
+**Query with `LIMIT` and `OFFSET`**
```

Comment:
Do not use bold in place of headings. The h1 will be set by the topic title, so the next one down in the hierarchy is h2.

```
# h1
## h2
### h3
```